README.md

```diff
@@ -25,6 +29,20 @@
     - Note that json-server does not validate your data, it’ll just save whatever you send it.
     - Visit [npmjs.com](https://www.npmjs.com/package/json-server) for more info
 - Use httpie (command line), Insomnia, Postman, or any other “ReST” testing application to POST some categories and products, using JSON, into your API so you have data to work with.
+- install `express`
+- import express : `const express = require('express');
+- Execute express: `const app = express(); `
+- Start server:
+```
+    /* give it a port number and optionally pass a function to call when app
+     starts listening on given port*/
+
+const port = process.env.PORT || 3000;
+app.listen(port, () => console.log(`Listening on port ${port}`));
+
+```
+
```

Comment:
I would like to see a note here about `node index.js` or whatever it needs to be to run this application